plugin/trino-bigquery/src/main/java/io/trino/plugin/bigquery/BigQueryQueryPageSource.java

```diff
@@ -102,11 +103,14 @@ public BigQueryQueryPageSource(
 
     private static String buildSql(BigQueryTableHandle table, String projectId, List<String> columnNames, Optional<String> filter)
     {
+        String columns = columnNames.stream().map(column -> format("`%s`", column)).collect(joining(","));
```

Comment:
nit: We can use `BigQueryUtil::quote` method. Also, no need to build `columns` when `columnNames` is empty. 
```java
String projection = columnNames.isEmpty() ? "*" : columnNames.stream().map(BigQueryUtil::quote).collect(joining(","));
```